sormas-base/setup/sormas.properties

```diff
@@ -305,4 +305,16 @@ app.url=
 
 # JNDI name of the DEMIS adapter that is connected to this SORMAS instance. Specifies where additionally deployed modules are stored.
 # Example: java:global/sormas-demis-adapter/DemisLabResultsAdapter
-#interface.demis.jndiName=
\ No newline at end of file
+#interface.demis.jndiName=
+
+# Turn off theses properties to disable a tab in a Case view
```

Comment:
These properties are no longer needed since they were moved to the FeatureType enum